helpers.cpp

```diff
@@ -782,7 +789,11 @@ void simple_header_impl(std::ostream & s, T subject, bool sub = false)
         s << "unnamed";
         categorize(s, subject, true, true);
     }
-    s << "</title><meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">" << (sub ? "<base href=\"..\">" : "") << "<link rel=\"stylesheet\" href=\"style.css\"></head><body><h1>";
+    s << "</title><meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">" << (sub ? "<base href=\"..\">" : "");
+    
+    add_styles_and_scripts(s);
```

Comment:
jQuery is quite large, so it should only be included on pages that actually use it.